src/ppx/grfx/grfx_command.cpp

```diff
@@ -71,6 +71,10 @@ void CommandBuffer::BeginRendering(const grfx::RenderingInfo* pRenderingInfo)
 
     BeginRenderingImpl(pRenderingInfo);
     mDynamicRenderPassActive = true;
+    mRenderArea              = pRenderingInfo->renderArea;
+    mRenderTargetView        = pRenderingInfo->pRenderTargetViews;
```

Comment:
I think this is a bug to assign it like this, since pRenderingInfo might go out of scope by the time we use this array. But if we switch to vector this doesn't matter.

Reply:
Done.